starboard/evergreen/shared/launcher.py

```diff
@@ -271,6 +275,43 @@ def _StageTargetsAndContentsRaspi(self):
     os.makedirs(os.path.join(target_staging_dir, 'lib'))
     shutil.copy(target_binary_src, target_binary_dst)
 
+  def _StageTargetsAndContentsRdk(self):
+    """Stage targets and their contents for GN builds for RDK platforms."""
+    content_subdir = os.path.join('usr', 'share', 'cobalt')
+
+    # TODO(b/296633713):
+    # Target_name should be self.target_name. However because loader is
+    # hardcoded on RDK, i.e. it always loads "libcobalt.so" in path "cobalt",
+    # it has to use this name.
+    target_name = 'cobalt'
+
+    # Copy target content and binary.
+    target_install_path = os.path.join(self.out_directory, 'install')
+    target_staging_dir = os.path.join(self.staging_directory, 'content', 'app',
+                                      target_name)
+    os.makedirs(target_staging_dir)
+
+    # TODO(b/218889313): Reset the content path for the evergreen artifacts.
+    content_subdir = os.path.join('usr', 'share', 'cobalt')
+    target_content_src = os.path.join(target_install_path, content_subdir)
+    target_content_dst = os.path.join(target_staging_dir, 'content')
+    shutil.copytree(target_content_src, target_content_dst)
+
+    shlib_name = f'lib{self.target_name}'
+    shlib_name += '.lz4' if self.use_compressed_library else '.so'
```

Comment:
This snippet is irrelevant for RDK right? The loader is hardcoded to load `libcobalt.so` so the `lz4` extension wouldn't work.

Reply:
Removed the irrelevant 'lz4'.